src/notfication/notify.py

```diff
@@ -0,0 +1,28 @@
+from email.message import EmailMessage
+import smtplib
+import os
+import ssl
+# from dotenv import load_dotenv
+# load_dotenv()
+
+class Notify:
+    # __serviceAccEmail = os.getenv('SERVICE_EMAIL')
+    # __serviceAccPassword = os.getenv('SERVICE_PASSWORD')
+
+    _subject = " Something went wrong !!"
+
+    def email(errorMessage:str,to:list[str]):
```

Comment:
Let's rename the method to `sendEmail` to better specify it's functionality.\
I know that we will only be using this class for error notifications but rather than just specifying the parameter name as `errorMessage`, let's make it `message`  to be a little more generic

Reply:
I know that we will only be using this class for error notifications but rather than specifying the parameter name as `errorMessage`, let's make it `message`  to be a little more generic

Reply:
Also, I don't see a `self` parameter in the method.\
Is this a static method?\
If so please add `@staticmethod` decorator, else add a `self` parameter in the function to make it correct.

Reply:
```python
def sendEmail(to:list[str], subject: str, message: str):
```

Something on these lines...

Reply:
ok